config_rb & knife exec updates for user & user keys global endpoints #2240 (#1781)
chef/chef-workstation: docs-chef-io/content/workstation/config_rb.md

  chef_server_url 'https://localhost/organizations/ORG_NAME'
  ```

  {{< note >}}

>   If changes need to be made to any global end points like user or user keys, add `--server-url` flag pointing to `https://localhost/`.

Reviewer: Maybe something like this.
```suggestion
  If changes need to be made to any global end points like user or user keys, use [`knife exec`]({{< relref "workstation/knife_exec" >}}) with the `--server-url` flag to set `chef_server_url` to `https://localhost/`.
```

>   If changes need to be made to any global end points like user or user keys, update `chef_server_url` in `knife.rb` to `https://localhost/` instead of `https://localhost/organizations/ORG_NAME`.

Reviewer 2: We probably want to suggest the `--server-url` flag here as well so people don't think they need to edit their config.rb for these operations

Author: Done.

  chef_server_url 'https://localhost/organizations/ORG_NAME'
  ```

  {{< note >}}
  If changes need to be made to any global end points like user or user keys, update `chef_server_url` in `knife.rb` to `https://localhost/` instead of `https://localhost/organizations/ORG_NAME`.
  {{< /note >}}

> 

Reviewer 2: Do we want all the extra spaces here?

Reviewer: Deleted
